nuxeo-distribution/nuxeo-launcher/src/main/java/org/nuxeo/launcher/NuxeoLauncher.java

```diff
@@ -565,6 +572,8 @@ protected boolean commandRequiresNoGUI() {
             + "        nuxeoctl mp-add|mp-install|mp-uninstall|mp-remove|mp-set|mp-request [command parameters] [-d [<categories>]|-q|--clid <arg>|--xml|--json|--nodeps|--relax <true|false|yes|no|ask>|--accept <true|false|yes|no|ask>|-s|-im]\n\n"
             + "        nuxeoctl register [<username> [<project> [<type> <description>] [<pwd>]]]\n\n"
             + "        nuxeoctl register-trial [<email> <company> <project> <description> [<pwd>]]\n\n"
+            + "        nuxeoctl register --clid <arg>\n\n"
+            + "        nuxeoctl register --renew\n\n"
```

Comment:
Either merge both into `nuxeoctl register [--renew] [--clid <file>]\n\n` and move up two lines before (for readability and preferred usage)?
Else the second usage (`renew`) is missing the optional value: `nuxeoctl register --renew [--clid <file>]`
And it may be worth to provide more detailed synopsis samples as for `nuxeoctl config` which has a description: it is not clear what can be done with the register command.